### PR TITLE
Use requestAnimationFrame for FUOC-less resolution

### LINKED ISSUE #4: requestAnimationFrame instead of setInterval
I used element-ready for https://github.com/sindresorhus/refined-github/pull/441 but it's not fast enough. Here's a comparison with the simple [rAF-based solution](https://github.com/bfred-it/github-clean-feed/blob/a52c08473630cd911137545b16769c32e975ca5e/src/content.js#L194-L202) I used in `github-clean-feed` and how that compares with element-ready's `setInterval`

```js
const domReady = new Promise(resolve => {
	(function check() {
		if (document.querySelector('.ajax-pagination-form')) {
			resolve();
		} else {
			requestAnimationFrame(check);
		}
	})();
});
```

reload screencast

⬆️ I'm refreshing the page. Notice the feed being instantly changed by `github-clean-feed`, but the Trending link appearing after the first paint.

`document.querySelector` is not that CPU-intensive, so running it _on every frame until domready_ is not that much to ask. Perhaps it can be an option. Either way I hope to see the change in RefinedGH

### PR DESCRIPTION
**Pull request by bfred-it**
Closes #4

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import m from './';
 
 global.document = jsdom.jsdom();
 global.window = document.defaultView;
+global.requestAnimationFrame = fn => setTimeout(fn, 16);
```
Comment (kevva) [inline test.js]: Is `16` arbitrary or why `16`?

**Comment by kevva** [comment]
:+1: lgtm

File: test.js (hunk 1 above)
Comment (bfred-it) [inline test.js]: rAF tries to match the monitor's refresh rate, which is generally 60hz.

100/60=16.66666666666

Generally speaking a frame is considered 16ms long.

Comment (kevva) [inline test.js]: Fair enough 👍

**Pull request merged**